install.sh

```diff
@@ -90,106 +87,121 @@ function curl () {
   # the 'curl: (7) Couldn't connect to server' error, a for loop is used
   # here.
   for (( i = 0; i < 4; i++ )); do
-    # force TLS as we know GitLab supports it
-    env -u LD_LIBRARY_PATH ${CURL} --ssl-reqd --tlsv1.2 -C - "${@}" && return 0
-    echo_info "Retrying, $((3-i)) retries left."
+    # Force TLS as we know GitLab supports it
+    if [ "$(env -u LD_LIBRARY_PATH "${CURL}" --ssl-reqd --tlsv1.2 -C - "${@}")" ]; then
+      echo_info "Retrying, $((3-i)) retries left."
+    else
+      return 0
+    fi
   done
   # The download failed if we're still here.
   echo_error "Download failed :/ Please check your network settings."
   return 1
 }
 
-case "${ARCH}" in
-"i686"|"x86_64"|"armv7l"|"aarch64")
-  LIB_SUFFIX=
-  # See https://superuser.com/a/1369875
-  # If /bin/bash is 64-bit, then set LIB_SUFFIX, as this is true on both
-  # x86_64 and aarch64 userspace
-  # shellcheck disable=SC2046
-  [ $(od -An -t x1 -j 4 -N 1  /bin/bash) == 02 ] && LIB_SUFFIX='64'
-  if [[ $LIB_SUFFIX == '64' ]] && [[ $ARCH == 'aarch64' ]]; then
-    echo_error "Your device is not supported by Chromebrew yet :/"
-    exit 1
-  fi
-  ;;
-*)
+# See https://superuser.com/a/1369875
+# If /bin/bash is 64-bit, then set LIB_SUFFIX, as this is true on both
+# x86_64 and aarch64 userspace
+# shellcheck disable=SC2046
+[ $(od -An -t x1 -j 4 -N 1  /bin/bash) == 02 ] && LIB_SUFFIX='64'
+if [[ $LIB_SUFFIX == '64' ]] && [[ $ARCH == 'aarch64' ]]; then
   echo_error "Your device is not supported by Chromebrew yet :/"
   exit 1
-  ;;
-esac
+fi
 
-echo_info "\n\nDoing initial setup for install in ${CREW_PREFIX}."
-echo_info "This may take a while if there are preexisting files in ${CREW_PREFIX}...\n"
+# Delete /usr/local/{var,local} symlinks which exist on some Chromium OS distros.
+if [ "${CREW_PREFIX}" = "/usr/local" ]; then
+  [ -L ${CREW_PREFIX}/var ] && sudo rm -f "${CREW_PREFIX}/var"
+  [ -L ${CREW_PREFIX}/local ] && sudo rm -f "${CREW_PREFIX}/local"
+fi
 
-# This will allow things to work without sudo.
-crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
-for folder in $crew_folders
-do
-  if [ -d "${CREW_PREFIX}"/"${folder}" ]; then
-    echo_intra "Resetting ownership of ${CREW_PREFIX}/${folder}"
-    sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"${folder}"
+if [ -d ${CREW_PREFIX} ]; then
+  if [ "$(ls -A ${CREW_PREFIX})" ]; then
+    echo_error "${CREW_PREFIX} is not empty, would you like it to be cleared?"
+    echo_info "This will delete ALL files in ${CREW_PREFIX}!"
+    echo_info "Continue?"
+    select continue in "Yes" "No"; do
```

Comment:
`No` not needed here.

Reply:
I felt it would be a little clearer for users if they had a `No` option, as even though any input other than `1` will exit the script, they might not know that.

Reply:
I suppose it's needed to provide the options.

Reply:
Yeah.